webapp/src/app/pages/landing-page/landing-page.component.ts

```
@@ -90,11 +90,11 @@
           this.Users = [];
 
           this.Users = course.users;
-
+          console.log(' this.Userss: ', this.Users); 
           let userModerator = this.Users.find(
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'userModerator' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)